### PR TITLE
page "Accept Terms and Conditions

### PR DESCRIPTION
**Pull request by GabrielaRsanches**

### CODE REVIEW

File: apps/atqr/web/src/pages/termsAndCondition/termsAndCondition.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```
Comment (albnunes) [inline apps/atqr/web/src/pages/termsAndCondition/termsAndCondition.html]: Use pug files.

**Review by albnunes — DISMISSED** [review]

File: apps/atqr/web/src/pages/termsAndCondition/termsAndCondition.html (hunk 1 above)
Comment (rairezende) [reply apps/atqr/web/src/pages/termsAndCondition/termsAndCondition.html]: @albnunes @yfernandes I'm taking care of this one

File: apps/atqr/web/src/pages/termsAndCondition/termsConditions.pug [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    meta(charset='utf-8')
+    meta(http-equiv='x-ua-compatible' content='ie=edge')
+    title ATQR - AJOELHOU TEM QUE REZAR
+    meta(name='robots' content='noindex, follow')
+    meta(name='description' content='')
+    meta(name='viewport' content='width=device-width, initial-scale=1, shrink-to-fit=no')
+
+
+    //- Favicon
+    link(rel='shortcut icon' type='image/x-icon' href='../../assets/images/favicon.png')
```
Comment (albnunes) [inline apps/atqr/web/src/pages/termsAndCondition/termsConditions.pug]: These imports can be made by a mixin that exists in the index folder

File: apps/atqr/web/src/pages/challenge/challenge.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```
Comment (albnunes) [inline apps/atqr/web/src/pages/challenge/challenge.js]: this file can't be here. exclude

File: apps/atqr/web/src/pages/termsAndCondition/termsConditions.pug [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+        button(class='btn btn-primary' type='submit') Aceitar termos e condições
+
+    //- Page Banner End
+    //- button(class='accept')
+
+  script(src='../../assets/js/vendor/modernizr-3.11.7.min.js')
+
+  script(src='../../assets/js/all.js')
+  script(src='../../assets/js/back-to-top.js')
```
Comment (albnunes) [inline apps/atqr/web/src/pages/termsAndCondition/termsConditions.pug]: same comment above

**Review by albnunes — CHANGES_REQUESTED** [review]
Make the requested changes

**Review by albnunes — DISMISSED** [review]

**Review by albnunes — DISMISSED** [review]
it's ok

**Pull request closed without merging**